.gitignore

```diff
@@ -21,4 +21,8 @@ yarn-error.log
 .vscode/
 
 # Ignore password file
-password.txt 
+password.txt
+
+# go dependancy
+/go.mod
```

Comment:
? , why add this

Reply:
added some go support for dependancies that added these. didn't want to push them